lib/client/api.go

```diff
@@ -2457,7 +2461,7 @@ func (tc *TeleportClient) PingAndShowMOTD(ctx context.Context) (*webclient.PingR
 // The returned Key should typically be passed to ActivateKey in order to
 // update local agent state.
 //
-func (tc *TeleportClient) Login(ctx context.Context) (*Key, error) {
+func (tc *TeleportClient) Login(ctx context.Context, pwdless bool) (*Key, error) {
```

Comment:
Would it make sense to have `pwdless` as property of `TeleportClient`?

Reply:
Done.